prometheus.yml

```diff
@@ -0,0 +1,20 @@
+# my global config
+global:
+  scrape_interval:     15s # Set the scrape interval to every 15 seconds. Default is every 1 minute.
+  evaluation_interval: 15s # Evaluate rules every 15 seconds. The default is every 1 minute.
+
+# A scrape configuration containing exactly one endpoint to scrape:
+# Here it's Prometheus itself.
+scrape_configs:
+  # The job name is added as a label `job=<job_name>` to any timeseries scraped from this config.
+  - job_name: 'FHIRQUERY'
+    # metrics_path defaults to '/metrics'
+    # scheme defaults to 'http'.
+    static_configs:
+    - targets: ['endpoint_querier_1:8443']
+
+remote_write:
+    - url: "http://prometheus_postgres_adapter:9201/write"
+remote_read:
+    - url: "http://prometheus_postgres_adapter:9201/read"
+
```

Comment:
I don't think this is the task to do it, but a task for next sprint is to figure out what our directory structure for the sub projects and for this top level should be. It's weird to me to have `prometheus.yml` at the top level.

Reply:
added https://oncprojectracking.healthit.gov/support/browse/LANTERN-87